src/tests/functional/plugin/shared/include/snippets/edge_replace.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/base/snippets_test_utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+typedef std::tuple<
+        InputShape,                  // Input 0 Shape
+        InputShape,                  // Input 1 Shape
+        InputShape,                  // Input 2 Shape
+        InputShape,                  // Input 3 Shape
+        InputShape,                  // Input 4 Shape
+        InputShape,                  // Input 5 Shape
+        InputShape,                  // Input 6 Shape
+        InputShape,                  // Input 7 Shape
```

Comment:
Do we really need the separate 8 shapes? :smile: Can we just use `std::vector`?

Moreover, you can use `ov::PartialShape` insetad of `InputShape` to use `init_input_shapes(static_partial_shapes_to_test_representation(input_shapes));` in `SetUp()`. What do you think?

Reply:
yeap, the number of input is reduced with a simplified reproduceble use case, and partial shape is used now. Thank you!